http/handler_store.go

```diff
@@ -411,6 +449,39 @@ func (h *storeHandler) bindRoutes(router *Router) {
 	collectionDescribe.AddResponse(200, collectionsResponse)
 	collectionDescribe.Responses["400"] = errorResponse
 
+	schemaNameQueryParam := openapi3.NewQueryParameter("name").
+		WithDescription("Schema name").
+		WithSchema(openapi3.NewStringSchema())
+	schemaSchemaRootQueryParam := openapi3.NewQueryParameter("root").
+		WithDescription("Schema root").
+		WithSchema(openapi3.NewStringSchema())
+	schemaVersionIDQueryParam := openapi3.NewQueryParameter("version_id").
+		WithDescription("Schema version id").
+		WithSchema(openapi3.NewStringSchema())
+
+	schemasSchema := openapi3.NewArraySchema()
+	schemasSchema.Items = schemaSchema
+
+	schemaResponseSchema := openapi3.NewOneOfSchema()
+	schemaResponseSchema.OneOf = openapi3.SchemaRefs{
+		schemaSchema,
+		openapi3.NewSchemaRef("", schemasSchema),
+	}
+
+	schemaResponse := openapi3.NewResponse().
+		WithDescription("Schema(s) with matching name, schema id, or version id.").
+		WithJSONSchema(schemaResponseSchema)
+
+	schemaDescribe := openapi3.NewOperation()
+	schemaDescribe.OperationID = "schema_describe"
+	schemaDescribe.Description = "Introspect schema(s) by name, schema root, or version id."
+	schemaDescribe.Tags = []string{"schema"}
+	schemaDescribe.AddParameter(schemaNameQueryParam)
+	schemaDescribe.AddParameter(schemaSchemaRootQueryParam)
+	schemaDescribe.AddParameter(schemaVersionIDQueryParam)
+	schemaDescribe.AddResponse(200, schemaResponse)
+	schemaDescribe.Responses["400"] = errorResponse
```

Comment:
Thanks for keeping these in sync!

Reply:
Thanks for making it simple enough to do so :)